src/openApiClientJs/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "joanie-openapi-client-ts",
+  "version": "1.0",
+  "description": "Tool to generate Typescript api client for joanie",
+  "scripts": {
+    "generate:api:client:local": "./scripts/openapi-typescript-codegen/generate_api_client_local.sh $1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openfun/joanie.git"
+  },
+  "author": {
+    "name": "Open FUN (France Université Numérique)",
+    "email": "fun.dev@fun-mooc.fr"
+  },
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/openfun/joanie/issues"
+  },
+  "homepage": "https://github.com/openfun/joanie#readme",
+  "devDependencies": {
+    "openapi-typescript-codegen": "^0.23.0"
```

We always pin dependencies version. Renovate bot is in charge to update them each week.
```suggestion
    "openapi-typescript-codegen": "0.23.0"
```